tools/build_config/Jenkinsfile

```diff
@@ -252,26 +272,31 @@ pipeline {
             }
 
           } else {
+            def herbert_ref = get_herbert_ref_from_labels(env.PR_LABELS,
```

Comment:
the documentation for this function needs to note that it will return an error string if multiple labels are found.